tools/fuzz/fuzz.cpp

```diff
@@ -75,9 +75,8 @@ USAGE: %s [options] <input.spv> -o <output.spv>
 USAGE: %s [options] <input.spv> -o <output.spv> \
   --shrink=<input.transformations> -- <interestingness_function> <args...>
 
```

Comment:
I think it would be clearer to say:

The SPIR-V binary is read from <input[.ext]>, e.g. 'shader.spv', or 'shader'.  If <input.facts> ('shader.facts' in both example cases) is also present, facts about the SPIR-V binary are read from this file.

But perhaps this is a bit too nit-picky.  What do you think?

Reply:
I personally find it easier to use input.spv throughout as it gives me a reminder of what the input is. I think the use of the .spv extension should appear to be almost required, but we should support other extensions for convenience.

Reply:
That's fine by me.